github/files/drupal7/scripts/devops/reinstall/post_settings.yml

```diff
@@ -23,6 +23,14 @@
   when: item.status == true
   with_items: env.post_settings
 
+- name: Finding a domain module
+  shell: 'find . -name domain -type d | grep contrib | cut -d. -f2 || true'
+  register: found
```

Comment:
variables should be self describing.
Like ```drupal_domain_access_state```
Because CIBox is something much more bigger that a small part of it named drupal